tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -157,6 +176,9 @@ def _test_fsx_lustre(
     _test_export_path(remote_command_executor, mount_dir, bucket_name, region)
     _test_auto_import(auto_import_policy, remote_command_executor, mount_dir, bucket_name, region)
     _test_data_repository_task(remote_command_executor, mount_dir, bucket_name, fsx_fs_id, region)
+    _test_storage_capacity(remote_command_executor, mount_dir, storage_capacity)
```

Comment:
We will want to include `_test_storage_capacity` for our tests that are specifying, but currently not testing storage capacity, like `test_fsx_lustre`.

Please double check for all existing tests

Reply:
Done